src/backend/JasmineGraphBackend.cpp

```diff
@@ -140,3 +167,9 @@ int JasmineGraphBackend::run() {
         pthread_join(threadA[i], NULL);
     }
 }
+
+void JasmineGraphBackend::updateWorkerStatus(std::string workerIP, std::string workerPort, SQLiteDBInterface sqLiteDbInterface) {
```

Comment:
Since this function's body is two lines of code shall we directly put the code in the function body in the code which does the function call?

Reply:
Then we need not declare the function.

Reply:
will move the code to the body

Reply:
Ok